src/testModel.ts

```diff
@@ -638,10 +635,15 @@ export class TestModel extends DisposableBase {
       for (const include of watch.include) {
         if (!include.uri)
           continue;
-        filesToWatch.add(include.uri.fsPath);
+        filesToWatch.add(uriToPath(include.uri));
       }
     }
-    await this._playwrightTest.watchFiles([...filesToWatch]);
+
+    // Playwright expects uppercase drive letter on Windows.
+    let files = [...filesToWatch];
+    if (process.platform === 'win32')
+      files = files.map(file => file[0].toUpperCase() + file.substring(1));
```

Comment:
Seeing this not next to uriToPath is unexpected.

Reply:
Oops, I forgot to remove this code. It's not needed, because now all the paths have uppercase drive letter already.